test/test_custom_ops.py

```diff
@@ -19,9 +19,11 @@
 from torch import Tensor
 from torch._custom_op.impl import custom_op, CustomOp, infer_schema
 from torch._utils_internal import get_file_path_2
+from torch.library import def_blackbox
```

Comment:
Taking bikeshedding. `def_opaque`? Or just call it `@custom_op`? (We expect the majority of custom ops that people write will be blackbox).

Reply:
opaque/traceable sound more like opposites than blackbox/traceable

Reply:
> Or just call it @custom_op? (We expect the majority of custom ops that people write will be blackbox).

I like this idea, maybe custom_op and traceable_custom_op? (Maybe i'm biased because we've been referring to it as custom op all this time, but if someone does `@custom_op` it feels clearer to me that we're creating a custom op. )

Reply:
@soulitzer the general problem is that no all custom ops are opaque. If we want this to be clearer we could call the decorator opaque_op/traceable_op or blackbox_op/traceable_op instead of def_blackbox/def_traceable